src/xrpld/consensus/Consensus.cpp

```diff
@@ -146,8 +147,9 @@ checkConsensusReached(
     }
 
     std::size_t currentPercentage = (agreeing * 100) / total;
-    CLOG(clog) << "currentPercentage: " << currentPercentage;
-    bool const ret = currentPercentage >= minConsensusPct;
+    CLOG(clog) << "currentPercentage: " << currentPercentage
+               << (stalled ? ", stalled" : "");
+    bool const ret = stalled || currentPercentage >= minConsensusPct;
```

Comment:
This is interesting. We only get `stalled` when every disputed transaction is unequivocally either under 20% or over 80% consensus (repectively "nay" or "yay") so there is no way to manipulate transactions which do not cut the threshold into the ledger. Would be good to put explanation to such effect in a comment, though.

Reply:
> This is interesting. We only get `stalled` when every disputed transaction is unequivocally either under 20% or over 80% consensus (repectively "nay" or "yay") so there is no way to manipulate transactions which do not cut the threshold into the ledger. Would be good to put explanation to such effect in a comment, though.

Comment added